hypertrace-graphql-spaces-schema/src/main/java/org/hypertrace/graphql/spaces/dao/SpaceConfigRulesResponseConverter.java

```diff
@@ -40,6 +41,16 @@
             Single.error(new IllegalArgumentException("Unable to convert rule creation response")));
   }
 
+  Single<Boolean> buildDeleteResponse() {
+    return Single.just(true);
```

Comment:
nothing in the response message like the deleted config rule?

Reply:
Nope - the UI doesn't need it, so didn't seem worth doing the fetch first (in the lower layer).